### PR TITLE
Fixed issue #6 - Add option to open link directly in browser

### PR DESCRIPTION
**Pull request by amankumarhq**
This PR addresses issue #6 for Hacktoberfest by adding the -o or --open option, allowing users to directly open the generated pull request URL in their default browser.

**Changes Made**
  Added a command-line argument to open the URL automatically using the open crate.

  **How to Test**
  _Run the command:_
          cargo run -- -r owner/repo -b my-feature-branch -d main -o

Thank you for considering this contribution!

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -167,29 +176,36 @@ fn main() {
...
-    // If no value found, check the config
     let value_from_config: Option<&str> = config.and_then(|cfg: &RcFile| {
         cfg.defaults
             .as_ref()
             .and_then(|d: &Defaults| d.values.get(key))
             .map(|s: &String| s.as_str())
     });
 
-    // Combine both options
     value_from_matches
```
Comment (ArunVenkata) [inline src/main.rs]: Don't remove comments that were already present

File: Cargo.toml [hunk 2]
```diff
@@ -16,8 +16,9 @@ clap = { version = "4.0", features = ["derive"] }
 config = "0.13"
 git2 = "0.19"
 percent-encoding = "2.3.1"
-
+# Add this dependency for opening URLs in the default browser
+webbrowser = "0.8"
```
Comment (ArunVenkata) [inline Cargo.toml]: Was there any reason for the package version to be set to 0.8 ?

If not, please update it to the latest version

**Review by ArunVenkata — CHANGES_REQUESTED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -16,8 +16,9 @@ clap = { version = "4.0", features = ["derive"] }
 config = "0.13"
 git2 = "0.19"
 percent-encoding = "2.3.1"
-
+# Add this dependency for opening URLs in the default browser
```
Comment (ArunVenkata) [inline Cargo.toml]:
```suggestion
```

File: Cargo.toml (hunk 2 above)
Comment (amankumarhq) [reply Cargo.toml]: updated it to the latest version now

File: src/main.rs (hunk 1 above)
Comment (amankumarhq) [reply src/main.rs]: Added the comments back

**Comment by amankumarhq** [comment]
@ArunVenkata Thank you for the feedback. I have made the changes. Please have a look

**Review by ArunVenkata — APPROVED** [review]

**Comment by ArunVenkata** [comment]
Merged, thank you for your contribution!

**Pull request merged**